src/SFA.DAS.AssessorService.EpaoImporter/InfrastructureServices/TokenService.cs

```diff
@@ -15,6 +15,9 @@ public TokenService(IWebConfiguration configuration)
 
         public string GetToken()
         {
+            if(string.Equals("LOCAL", ConfigurationHelper.GetEnvironmentName()))
```

Comment:
This is different to all the other places which is use DevelopmentEnvironment

Reply:
It's to do with it being a Function app and I couldn't find an injected IHostingEnvironment